orttraining/orttraining/python/training/ortmodule/_experimental/_load_config_from_json.py

```diff
@@ -0,0 +1,146 @@
+
+def load_enable_grad_acc_optimization(ortmodule_config_accessor, data):
+    """Load EnableGradAccOptimization from json file onto ORTModule"""
+
+    enable_grad_acc_optimization_str = 'EnableGradAccOptimization'
+    if not hasattr(data, enable_grad_acc_optimization_str):
+        return
+    print(f"Found keyword {enable_grad_acc_optimization_str} in json. Loading attributes from file.")
+
+    assert isinstance(data.EnableGradAccOptimization, bool), f"{enable_grad_acc_optimization_str} must be a boolean"
+    ortmodule_config_accessor._enable_grad_acc_optimization = data.EnableGradAccOptimization
+
+def load_run_symbolic_shape_infer(ortmodule_config_accessor, data):
+    """Load RunSymbolicShapeInference from json file onto ORTModule"""
+
+    run_symbolic_shape_inference_str = 'RunSymbolicShapeInference'
+    if not hasattr(data, run_symbolic_shape_inference_str):
+        return
+    print(f"Found keyword {run_symbolic_shape_inference_str} in json. Loading attributes from file.")
+
+    assert isinstance(data.RunSymbolicShapeInference, bool), f"{run_symbolic_shape_inference_str} must be a boolean"
+    ortmodule_config_accessor._run_symbolic_shape_infer = data.RunSymbolicShapeInference
+
+def load_use_static_shape(ortmodule_config_accessor, data):
+    """Load UseStaticShape from json file onto ORTModule"""
+
+    use_state_shape_str = 'UseStaticShape'
+    if not hasattr(data, use_state_shape_str):
+        return
+    print(f"Found keyword {use_state_shape_str} in json. Loading attributes from file.")
+
+    assert isinstance(data.UseStaticShape, bool), f"{use_state_shape_str} must be a boolean"
+    ortmodule_config_accessor._use_static_shape = data.UseStaticShape
+
+def load_skip_check(ortmodule_config_accessor, data):
+    """Load SkipCheck from json file onto ORTModule"""
+
+    skip_check_str = 'SkipCheck'
+    if not hasattr(data, skip_check_str):
+        return
+    print(f"Found keyword {skip_check_str} in json. Loading attributes from file.")
+
+    skip_check = reduce(lambda x, y: x|y, [_SkipCheck[name] for name in data.SkipCheck])
+    if skip_check.value > 0:
+        ortmodule_config_accessor._skip_check = skip_check
+
+def load_debug_options(ortmodule_config_accessor, data):
+    """Load DebugOptions from json file onto ORTModule"""
+
+    debug_options_str = 'DebugOptions'
+    if not hasattr(data, debug_options_str):
+        return
+    print(f"Found keyword {debug_options_str} in json. Loading attributes from file.")
+
+
+    log_level = LogLevel.WARNING
+    if hasattr(data.DebugOptions, 'LogLevel'):
+        log_level = LogLevel[data.DebugOptions.LogLevel]
+
+    save_onnx = False
+    onnx_prefix = ''
+    if hasattr(data.DebugOptions, 'SaveONNX'):
+        save_onnx = data.DebugOptions.SaveONNX
+        onnx_prefix = data.DebugOptions.ONNXPrefix
+
+    if hasattr(data.DebugOptions, 'SaveONNXPath'):
+        os.environ["ORTMODULE_SAVE_ONNX_PATH"] = data.DebugOptions.SaveONNXPath
```

Comment:
Using the global from DebugOptions's class would make the implementation more robust to intentional changes